airflow/executors/base_executor.py

```diff
@@ -355,14 +355,15 @@ def execute_async(
         """
         raise NotImplementedError()
 
-    def get_task_log(self, ti: TaskInstance, log: str = "") -> None | str | tuple[str, dict[str, bool]]:
+    def get_task_log(self, ti: TaskInstance) -> tuple[list[str], list[str]]:
```

Comment:
FileTaskHandler has this pattern that would break after this signature change:

```python
response = executor.get_task_log(ti=ti)
if response:
    ...
```

Also need to fix tests that currently use the removed `log` parameter.

Reply:
this bit
```
response = executor.get_task_log(ti=ti)
if response:
    executor_messages, executor_logs = response
```

it's a remnant from when if `get_task_log` not implemented then would return None, but i have since updated baseexecutor implementation to return `[], []` so that bit of defensiveness is not strictly necessary, though it doesn't hurt.

i have fixed tests, hopefully not too many more iterations.